config/initializers/react_on_rails.rb

```diff
@@ -18,10 +18,21 @@ def self.custom_context(view_context)
   # Define the files for we need to check for webpack compilation when running tests.
   config.webpack_generated_files = %w( app-bundle.js vendor-bundle.js server-bundle.js )
 
-  # The server bundle is a single file for all server rendering of components.
-  # If you are not using server rendering `(prerender: true)`, set this to "".
+  # This is the file used for server rendering of React when using `(prerender: true)`
+  # If you are never using server rendering, you may set this to "".
+  # If you are using the same file for client and server rendering, having this set probably does
+  # not affect performance.
   config.server_bundle_js_file = "server-bundle.js"
 
+  # If you are using the ReactOnRails::TestHelper.configure_rspec_to_compile_assets(config)
+  # with rspec then this controls what npm command is run
+  # to automatically refresh your webpack assets on every test run.
+  config.npm_build_test_command = "npm run build:test"
+
+  # This configures the script to run to build the production assets by webpack. Set this to nil
+  # if you don't want react_on_rails building this file for you.
+  config.npm_build_production_command = nil #"npm run build:production"
```

Comment:
Mental note: if and when we start to use "npm run build:production", we need to check what can be done to duplicate asset handling in our webpack plugin: customfileloader